extractor/main.go

```diff
@@ -58,7 +58,7 @@ func main() {
 
 	// Run the command
 	if err := cmd.ParseAndRun(context.Background(), os.Args[1:]); err != nil {
-		fmt.Fprintf(os.Stderr, "%+v", err)
+		_, _ = fmt.Fprintf(os.Stderr, "%+v", err)
```

Comment:
that is to make the linter happy?

Reply:
Yup

Reply:
This will do the same:

```suggestion
		println(err)
```

Reply:
even better, use https://go.dev/blog/slog

Reply:
Implemented slog: [b4ebcf6](https://github.com/gnolang/tx-exports/pull/12/commits/b4ebcf677ce21f9ef4ee3c19443e5f98da358928)